Misc cleanup

rshipp opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

### Description

Use fieldset in place of `industrySelectorContainer`, rename `questionFieldset` to `fieldset`, remove fileUpload and tooltips, misc small fixes.

### Motivation and Context

Task: https://app.asana.com/0/1174274412967132/1199178251531117/f

<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open ticket or issue, please provide the link here. -->

### Types of changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist:

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask around. -->

- [ ] My code follows the code style of this project.
- [ ] I have added neccessary labels
- [ ] My PR title follows the convention in the guide
- [ ] I have assigned myself to the PR
- [ ] I have requested reviews from team members
- [ ] I have created a task for reviewers on PM tool
- [ ] All new and existing tests passed.

jessouyangbraven reviewed (approved):
The test failures are weird, but I'm assuming it's branch related and will go away after rebasing?

Thanks for handling the industry selector stuff as well. I didn't realize until today's stand-up that the fieldsets were meant for accessibility and that I should have built it that way in the first place.

rshipp commented:
I also removed our custom ImageLink plugin and replaced it with the official LinkImage plugin, fixing https://app.asana.com/0/1170776727341290/1173552178131083/f
